README.md

```diff
@@ -9,27 +9,15 @@ A core foundation for rich Web applications.
 
 ## Overview
 
-When creating a Web app, you need a foundation on which to build. There are
-a lot of disparate technologies out there that can be brought together into
-a cohesive whole to make this happen. The trouble is in finding, vetting,
-testing, and combining these technologies -- all of which needs to happen
-before you can begin to make serious progress on your own project.
```

Comment:
I think this is helpful / connective / empathetic text explaining the purpose of Bedrock. If it can be said more concisely that's fine, but I don't want to lose it.

Reply:
Brought this back in 1694891